contrib/luarc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "runtime.version": "LuaJIT",
+    "diagnostics": {
+        "enable": true,
+        "globals": ["vim"]
```

Comment:
Should we also add the test methods like `describe`, `setup`, `teardown` and so on?

Reply:
I think that came up before ("how do I make `lua-language-server` not complain so much about the test scripts?").